src/components/PlanRoom/RoomTables/Types.tsx

```diff
@@ -0,0 +1,22 @@
+import { ReactNode } from 'react';
+
+export type TypeItem = {
```

Comment:
`TableType` and `TypeItem` have many common properties. 
You should extract them to separate type and use [intersection](https://levelup.gitconnected.com/typescript-advanced-types-union-and-intersection-types-9283046d7859).

Reply:
these types will be more integrated